workspaces/lightspeed/plugins/lightspeed-backend/src/service/router.ts

```diff
@@ -64,10 +64,12 @@ export async function createRouter(
 
   const authorizer = userPermissionAuthorization(permissions);
 
-  // Middleware proxy to exclude /v1/query
+  // Middleware proxy to exclude passthroughPaths
   router.use('/v1', async (req, res, next) => {
-    if (req.path === '/query') {
-      return next(); // This will skip proxying and go to /v1/query endpoint
+    const passthroughPaths = ['/query', '/feedback', '/feedback/status'];
```

Comment:
shouldn't `/feedback/status` be covered by `/feedback` already?

Reply:
I think it wasn't covered, let me check again.

Reply:
`.includes` does an exact match so it wouldn't work, so now I have modified the code `passthroughPaths.some(path => req.path.startsWith(path))` so `/feedback` will cover both /feedback and /feedback/status endpoints.